config/initializers/devise.rb

```diff
@@ -205,7 +205,9 @@
   # config.rememberable_options = {}
 
   # ==> Configuration for :validatable
-  # Range for password length.
+  # Range for password length. This is kept for use
+  # for admin users. Non-admin users have a higher minimum
+  # password length as defined by User.PASSWORD_LENGTH.
```

Comment:
We would strongly prefer relying on setting `config.password_length` to the new minimum everywhere. As far as we can tell, if we increase this (`config.password_length`) and an admin logs in, they won't be disrupted, so we can't really understand the explanation of "kept for use for admin users." In the unlikely event an admin decides on their own volition (or a new invite) to set a new password, it's fine if it has to be >6 chars.